src/rabbitmq.js

```diff
@@ -233,6 +214,36 @@ class RabbitMQ {
   }
 
   /**
+   * Validate publish params. Also add tid to job if one does not exist
+   * @param  {Strong} name:    string        Name of queue or exchange
+   * @param  {Object} content: Object        content Content to send.
+   * @return {Buffer}          Content to send in job
+   * @throws {Error} If not connected
+   * @throws {Error} If name is not a string
+   * @throws {Error} If content is not an object
+   */
+  _validatePublish (name: string, content: Object) {
```

Comment:
why do you need a name? are you just passing this because it's all the parameters for the publish methods? seems unnecessary...